### ISSUE TITLE
dogears-back and dogears-forward do nothing

### ISSUE THREAD

**Issue by Dapuva76** [alphapapa/dogears.el#5]
Just installed the latest version from melpa. I can see the list of dogears with dogears-list, but executing dogears-back and dogears-foward doesn't change between dogears. I would expect it to go back and forward through the list, but that doesn't happen.

**Comment by Dapuva76** [comment]
Ok I think I figured it out. There were conflicts *maybe* with evil-jump-backward and -forward.

**Comment by alphapapa** [comment]
If I had a nickle for every time an EVIL user said that something wasn't working...  ;)

**Comment by kings2u** [comment]
I am not an Evil user but I have experienced some inconsistency that might explain @Dapuva76's issue. 
I've noticed that when I press `dogears-back` enough times,  I will reliably arrive at the message "Already at oldest dogeared place." But if I press `dogears-forward` enough times to try to move past the most recently dogeared location, emacs will not signal an error telling me this. If I continue to press `dogears-forward` another three times (to no effect) and then press `dogears-back` once, point will not move. It won't move until I have pressed `dogears-back` another two times -- enough to offset the times I pressed `dogears-forward` to no effect.
